crates/ruma-common/src/events/poll/end.rs

```diff
@@ -68,6 +74,73 @@ impl PollEndEventContent {
             relates_to: Reference::new(poll_start_id),
         }
     }
+
+    /// Creates a new `PollEndEventContent` using the given poll and responses to generate the
+    /// results.
+    ///
+    /// It generates a default text representation of the results in english.
+    ///
+    /// This uses [`compute_poll_results()`] internally.
+    pub fn with_poll_results<'a>(
+        poll: &'a OriginalSyncPollStartEvent,
+        responses: impl IntoIterator<Item = &'a OriginalSyncPollResponseEvent>,
+    ) -> Self {
```

Comment:
Instead of `PollEndEventContent::with_poll_results(poll_start, responses)`, how about `poll_start.compile_result(responses)`? I think it would be more convenient to use.

Reply:
Sure